homeassistant/components/amberelectric/sensor.py

```diff
@@ -106,6 +106,7 @@ def extra_state_attributes(self) -> Mapping[str, Any] | None:
         data["renewables"] = round(interval.renewables)
         data["estimate"] = interval.estimate
         data["spike_status"] = interval.spike_status.value
+        data["descriptor"] = normalize_descriptor(interval.descriptor)
```

Comment:
Why do we add a state attribute here if we also add individual sensors?

Reply:
One of the use cases for this is to create lights that represent the price, as well as course grain automatons (ie, turn off when price is high, rather than turn off when price is 36c). If it is only an attribute, it would require a template, so I made it a first-class citizen.

Reply:
For more context: "high" isn't the same price threshold for all users, so using this sensor, they can map what is exactly happening in our app.

Reply:
It's added to the attributes for completeness (so it represents the whole object returned from the APi)

Reply:
I suggest we remove the added attributes as we have the sensors.